app/services/trainees/create_from_hesa.rb

```diff
@@ -13,6 +13,9 @@ def initialize(student_node:)
     def call
       trainee.assign_attributes(mapped_attributes)
       trainee.save!
+
+      create_degrees!
+
       trainee
       add_multiple_disability_text!
```

Comment:
This isn't part of your PR.. but should add_multiple_disability_text! be above trainee?